### ISSUE TITLE
represent bigints in a safe way

### ISSUE THREAD

**Issue by calvinmetcalf** [mafintosh/protocol-buffers#54]
it would be nice to be able to get ints that can't be represented as doubles as strings, buffers, or BN.js objects

**Comment by mafintosh** [comment]
what would be the best way to handle this in your opinion? an option that always returns int64's as strings?

**Comment by calvinmetcalf** [comment]
yeah that would work, also to take an int64 as a string would be great (as
that's why it came up for me)

**Comment by sshutovskyi** [comment]
this [PR](https://github.com/mafintosh/protocol-buffers/pull/63) attempts to solve the problem using bytebuffers package
